src/gui/generalsettings.cpp

```diff
@@ -80,6 +80,11 @@ GeneralSettings::GeneralSettings(QWidget *parent)
 
     _ui->crashreporterCheckBox->setVisible(Theme::instance()->withCrashReporter());
 
+    connect(_ui->moveToTrashCheckBox, &QCheckBox::toggled, this, [](bool checked) {
+        ConfigFile().setMoveToTrash(checked);
+        FolderMan::instance()->reloadSyncOptions();
+    });
```

Comment:
Hmm no strong opinion but wouldn't a "options changed" signal be Qt'er?

Reply:
This is about the same as done in `NetworkSettings::saveBWLimitSettings()` where a call is made to `FolderMan::instance()->setDirtyNetworkLimits();`

Reply:
setDirtyNetworkLimits is 10 years old :D 

I'd prefer a `Q_EMIT syncOptionsChanged()` signal.
and a 
```
connect(foo, &GeneralSettings::syncOptionsChanged, _engine, []{ setSyncOptions(loadSyncOptions());});
````
or something like that?